observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,73 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6_{context}"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3868[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements_{context}"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6_{context}"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now use Prometheus metrics and rely less on Loki for storage when using the Network Observability Operator. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6_{context}"]
+==== Custom metrics API
```

Comment:
<FLOW_METRICS_API_REF_LINK_HERE> is appearing in the preview.

Reply:
🤦‍♀️